packages/widgets/src/lib/components/InferenceWidget/widgets/ConversationalWidget/ConversationalWidget.svelte

```diff
@@ -82,7 +82,7 @@
 			updateUrl({ text: trimmedText });
 		}
 
-		if (!withModelLoading) {
+		if (!withModelLoading && messages[-1]?.role !== "user") {
```

Comment:
```suggestion
		if (!withModelLoading && messages.at(-1)?.role !== "user") {
```
JS not PY 😄

Reply:
Oh :disappointed: This [line](https://github.com/huggingface/huggingface.js/blob/a025e231c3cf0370134a541fcb4b8c22a15589e5/packages/widgets/src/lib/components/InferenceWidget/widgets/ConversationalWidget/ConversationalWidget.svelte#L155) made me think it was possible ^^

Reply:
Thanks for the tips though! :)